src/App.js

```diff
@@ -186,13 +186,30 @@ function App() {
   }, [theme]);
 
   const [checkingAPIs, setCheckingAPIs] = useState(true);
-
+  useEffect(() => {
+    setauth0works(isAuthenticated === true || isAuthenticated === false);
```

Comment:
If `auth0Works` doesn't really depend on the value of `isAuthenticated`, why are we even using a state and observing its change?